tests/ui/doc/doc.stderr

```diff
@@ -18,173 +18,197 @@ error: you should put `Foo::some_fun` between ticks in the documentation
 LL | /// Markdown is _weird_. I mean _really weird_. This /_ is ok. So is `_`. But not Foo::some_fun
    |                                                                                   ^^^^^^^^^^^^^
 
-error: you should put `a::global:path` between ticks in the documentation
-  --> $DIR/doc.rs:11:15
+error: you should put `::a::global:path` between ticks in the documentation
+  --> $DIR/doc.rs:11:13
    |
-LL | /// Here be ::a::global:path.
-   |               ^^^^^^^^^^^^^^
+LL | /// Here be ::a::global:path, and _::another::global::path_.  :: is not a path though.
+   |             ^^^^^^^^^^^^^^^^
+
+error: you should put `::another::global::path` between ticks in the documentation
+  --> $DIR/doc.rs:11:36
+   |
+LL | /// Here be ::a::global:path, and _::another::global::path_.  :: is not a path though.
+   |                                    ^^^^^^^^^^^^^^^^^^^^^^^
+
+error: you should put `::` between ticks in the documentation
+  --> $DIR/doc.rs:11:63
+   |
+LL | /// Here be ::a::global:path, and _::another::global::path_.  :: is not a path though.
+   |                                                               ^^
```

Comment:
The single `::` is now linted as a path. Do we want to keep it this way? In that case, it would be better to update the doc comment :upside_down_face:

Reply:
Darn, you're perfectly right to nitpick here, I had missed that in the last update! I'll fix it, it makes no sense to lint it IMO.

Reply:
Good, I sometimes feel like I'm to nitpicky :joy:

Reply:
Updated, thanks for the catch!